### ISSUE TITLE
nao: NVDA 2023.1 compatibility

### ISSUE THREAD

**Issue by surfer0627** [sharkboyto/nao#38]
Notes:

> this is a Add-on API compatibility breaking release. Add-ons will need to be re-tested and have their manifest updated.  
Once a year, NV Access may publish a backwards incompatible NVDA release, typically the first major release (year.1). Here, "backwards incompatible" refers to NVDA releases that make add-ons incompatible due to API changes. When this happens, authors must test their add-ons for compatibility and release updates accordingly.

* [NVDA Add-on API changes](https://groups.google.com/a/nvaccess.org/g/nvda-api?pli=1)
* [Download NVDA 2023.1beta1](https://www.nvaccess.org/files/nvda/releases/2023.1beta1/nvda_2023.1beta1.exe)
* [What’s new in this release of NVDA](https://www.nvaccess.org/files/nvda/releases/2023.1beta1/documentation/changes.html)
* [Add-ons and backwards incompatible NVDA releases](https://addons.nvda-project.org/addons/nvdacompat.en.html)

Thank you for your help.

**Comment by dreinn** [comment]
thank you for your reminder.
On a fast glance, Nao should be fully compatible with NVDA 2023.1.
Anyway we will check all code to be totally sure.



Thank you.

**Comment by surfer0627** [comment]
Thank you for the reply and contribution.

> On a fast glance, Nao should be fully compatible with NVDA 2023.1.

Starting with NVDA 2023.1 beta1, I got a message down below:

> Installation of Nao 2022.1.3 has been blocked. An updated version of this add-on is required, the minimum add-on API supported by this version of NVDA is 2023.1

Thank you for your kindly help.
